doc/architecture/decisions/0011-control-applicant-access-using-delius-team-caseload.md

```diff
@@ -0,0 +1,168 @@
+# 11. Control "applicant" access using Delius team caseload
+
+Date: 2023-01-23
+
+## Status
+
+Accepted
+
+## Context
+
+The service will manage all but one of its roles internally. This ADR concerns that one
+exception: the "applicant" role which is not modelled explicitly within the service, but
+is derived from caseload information from Delius.
+
+The Approved Premises service exposes personal data from NOMIS and from OASys and so
+it's necessary to restrict creating and viewing "applications" to probation officers with an
+appropriate caseload.
+
+
+
+### Overview of the service's roles
+
+#### Applicant
+**Role**: `APPLICANT`  
+**Description**: probation officer making application  
+**Authorised to**:
+- create an application subject to the restrictions on their Delius user account, with 
```

Comment:
Should we add something about continuing applications that they have made? Or do we want them to be able to continue applications their teammates have made? Though this could get messy.

Reply:
I think we've generally agreed that:

- the creator of a application can edit it until it's been submitted and then it's read-only -- apart from the ability to add clarification notes (the form of these post-submission exchanges between assessor and applicant is not finalised)
- collaborating on an application is out of scope, i.e. if I am struck by lightning halfway through creating an application then a colleague will need to create a new application from scratch